doc/kubernetes/modules/ROOT/pages/running/metrics/ispn_metrics.adoc

```diff
@@ -0,0 +1,242 @@
+
+| `vendor_statistics_approximate_entries_unique`
+| The approximate number of entries stored by the node, excluding copies.
+
+|===
+
+== Data Access
+
+The following metrics monitor each cache accesses, meaning reads, writes and removals.
+
+|===
+|Metric |Description
+
+| `vendor_statistics_read_write_ratio`
+| The read/write ratio.
+A value greater than 1.0 means the cache is more read-intensive.
+
+| `vendor_statistics_hit_ratio`
+| The percentages of reads that found the value.
+
+|===
+
+=== Stores
+
+Stores requests include operations like `put()` or `replace()`.
+With these metrics, it is possible to monitor the store request duration.
+
+|===
+|Metric |Description
+
+| `vendor_statistics_store_times_seconds_count`
+| The total number of store requests.
+
+| `vendor_statistics_store_times_seconds_sum`
+| The total duration of all store requests.
+
+|===
+
+NOTE: {histogram-note}
+
+=== Reads
+
+The read requests divides in 2 groups, hits and misses.
+If the read finds a value stored, it is a hit, and, if no value is found, it is a miss.
+These metrics allow to monitor the read request duration.
+
+|===
+|Metric |Description
+
+| `vendor_statistics_hit_times_seconds_count`
+| The total number of read hits requests.
+
+| `vendor_statistics_hit_times_seconds_sum`
+| The total duration of all read hits requests.
+
+| `vendor_statistics_miss_times_seconds_count`
+| The total number of read misses requests.
+
+| `vendor_statistics_miss_times_seconds_sum`
+| The total duration of all read misses requests.
+
+|===
+
+NOTE: {histogram-note}
+
+=== Removes
+
+Similar to cache reads, the remove requests divide in 2 groups.
+If the value is removed, it is a hit.
+If the value does not exist, it is a miss.
+These metrics allow to monitor the remove request duration.
+
+|===
+|Metric |Description
+
+| `vendor_statistics_remove_hit_times_seconds_count`
+| The total number of remove hits requests.
+
+| `vendor_statistics_remove_hit_times_seconds_sum`
+| The total duration of all remove hits requests.
+
+| `vendor_statistics_remove_miss_times_seconds_count`
+| The total number of remove misses requests.
+
+| `vendor_statistics_remove_miss_times_seconds_sum`
+| The total duration of all remove misses requests.
+
+|===
+
+NOTE: {histogram-note}
+
+== Locking
+
+Write operations hold the lock until the value is replicated in the local cluster and to the remote site.
+In a healthy cluster, this value should stable with little variance.
+Deadlocks may create temporary spikes.
+
+|===
+|Metric |Description
+
+| `vendor_lock_manager_number_of_locks_held`
+| The number of locks currently being held by this node.
+
+|===
+
+== Transactions
+
+{jdgserver_name} uses both One-Phase-Commit and Two-Phase-Commit protocols to complete a transaction.
+There metrics keep track of the duration for the operations.
+
+NOTE: The `PESSMISTIC` locking mode uses One-Phase-Commit and does not create commit requests.
```

Comment:
Please comment if you want to update this or keep it as is.

Reply:
I would keep it if it is ok for you. Users can still change the configuration via XML and it would be future proof if we change the defaults in some way.

Reply:
OK, then let's keep it. This is then resolved.